docs/sidebars.js

```diff
@@ -140,6 +140,7 @@ module.exports = {
           ],
         },
         "guides/integrate/logout",
+        "guides/integrate/event-api",
```

Comment:
I would suggest that we put it in Integrate>"Access ZITADEL APIs"

Reply:
I changed it